lib/fluent/plugin/out_azure-storage-append-blob.rb

```diff
@@ -115,10 +115,23 @@ def write(chunk)
           tmp.unlink
         end
       end
+
+      def container_exists?(container)
```

Comment:
Can you please add a test for this function?

Reply:
I added tests for this function.